hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/WALEntryStream.java

```diff
@@ -316,9 +316,8 @@ private boolean openNextLog() throws IOException {
     return false;
   }
 
-  private Path getArchivedLog(Path path) throws IOException {
+  Path getArchivedLog(Path path) throws IOException {
```

Comment:
Let's move this method to a util class so we do not need to change the modifier and also do not need to pass the WALEntryStream to the handleEofException?